runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -127,6 +127,9 @@ protected void startUp() throws Exception
 			updateInfoBoxState();
 		}
 
+		if (config.pointsMessage())
```

Comment:
Use brackets here

Reply:
Any specific reason? I'm used to doing this if there's just one line in the statement, so there's several cases where there are no brackets in the plugin.

Reply:
I am just following what I saw around the project, and mostly 1 line statements still have them, so, hmm, consistency is key I guess?

Reply:
I think the CONTRIBUTING doc specifies both the "all curlies go on new lines", and the "all `if` s require curly braces" rules. (if not, it *should*).

`if`s without braces are error-prone, since adding a second statement requires adding new braces, and failing to do so causes one of the two statements to run unconditionally.